tests/lib/journalctl.sh

```diff
@@ -65,3 +64,8 @@ get_journalctl_log_from_cursor(){
         journalctl "$@" --cursor "$cursor"
     fi
 }
+
+sync_journalctl_log(){
+    journalctl --flush || true
```

Comment:
When can those two commands fail and why should we ignore the failure?

Reply:
it is because depending on the system sometime --flush or --sync could not be supported